app/src/main/java/com/globant/pokemontcgapp/fragment/PokemonSupertypeFragment.kt

```diff
@@ -51,17 +55,30 @@ class PokemonSupertypeFragment : Fragment() {
 
     private fun showPokemonSupertypes(pokemonSupertypes: List<SecondaryTypes>) {
         binding.pokemonAlltypesLoading.visibility = View.GONE
-        binding.pokemonAlltypesRecyclerView.apply {
-            layoutManager =
-                GridLayoutManager(
-                    context,
-                    resources.configuration.getColumnsByOrientation(
-                        COLUMNS_PORTRAIT,
-                        COLUMNS_LANDSCAPE
-                    )
+        pokemonSupertypes.let {
+            val pokemonSupertypesAdapter = PokemonSecondaryTypesAdapter(pokemonSupertypes) { pokemonSupertypeElement ->
+                startActivity(
+                    context?.let { it1 ->
+                        PokemonCardListActivity.getIntent(it1).apply {
```

Comment:
create a method that return the intent with all the extras in the PokemonCardListActivity

Reply:
done, thanks